### ISSUE TITLE
Summary stats release report email needs to be more human-readable

### ISSUE THREAD

**Issue by ljwh2** [EBISPOT/goci#109]
The purpose of this email is to alert curators to manual errors in the sumstats release process, eg typo in folder name, curator forgot to check "full pvalue set" box.
Now that the folders are named with GCSTs only, this has become non-intuitive. The first author and PMID should be included in the email together with the folder names to allow curators to easily check that expected sumstats have been released.

**Comment by ljwh2** [comment]
@earlEBI Please review as to whether this issue is still relevant

**Comment by earlEBI** [comment]
I don't think this is important anymore. Folders are listed as renamed with author name and PMID once published. 
Typos in folder names and not ticking 'full p-value set' box are not really possible curator errors anymore.

**Comment by ljwh2** [comment]
@jdhayhurst tagging you for reference, I am moving this ticket to the icebox
